python/nano/tutorial/notebook/inference/pytorch/multi_instance_pytorch_inference.ipynb

```diff
@@ -38,9 +38,8 @@
    },
    "outputs": [],
    "source": [
-    "# for BigDL-Nano\n",
-    "!pip install --pre --upgrade bigdl-nano[pytorch] # install the nightly-bulit version\n",
-    "# !source bigdl-nano-init"
+    "!pip install --pre --upgrade bigdl-nano[pytorch,inference] # install the nightly-built version\n",
```

Comment:
I think we need to install `bigdl-nano[inference]` only when we want to use inc/openvino/onnx, only using multi-instance does not require `[inference]`

Reply:
In the [installation table](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/install.html) we provided for users, we only have options for training and inference. So I feel like it could be more user-friendly to let user install with `bigdl-nano[pytorch,inference]` for all the PyTorch Inference cases. So, I unify them here.